metadata.rb

```diff
@@ -1,14 +1,14 @@
-name             'maven'
-maintainer       'Chef Software, Inc.'
+name 'maven'
+maintainer 'Chef Software, Inc.'
 maintainer_email 'cookbooks@chef.io'
-license          'Apache 2.0'
-description      'Installs maven 2 or 3 and includes a maven resource.'
+license 'Apache 2.0'
+description 'Application cookbook which installs and configures Maven.'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '1.3.0'
+version '2.0.0'
 
-depends 'ark',  '~> 0.4'
-depends 'java', '~> 1.13'
-depends 'windows'
+depends 'libarchive', '~> 0.6'
+depends 'poise', '~> 2.2'
```

Comment:
It would seem that Java is an actual dependency of this cookbook, since you cannot do Maven things without it… right?